docs/admonitions.md

```diff
@@ -167,6 +167,7 @@ This is the body.
 To turn an admonition into a dropdown, add the `dropdown` {myst:directive}`admonition.class` to them.
 Dropdown admonitions use the `<details>` HTML element (meaning they also will work without Javascript!),
 and they can be helpful when including text that shouldn't immediately visible to your readers.
+To have the admonition be a dropdown, and also start open, add the {myst:directive}`admonition.open` option.
```

Comment:
```suggestion
To have a dropdown-style admonition start open, add the {myst:directive}`admonition.open` option.
```

Reply:
The `open` option doesn't add the `class` for dropdown, right?

Reply:
Perhaps we should add the `dropdown` class if `open` is defined?

Reply:
@rowanc1 should we warn the user if they don't define `class: dropdown` then?

Reply:
Yeah - I will add that. Right now it just doesn't do anything silently.